swarm/storage/resource_test.go

```diff
@@ -29,16 +27,22 @@ import (
 )
 
 var (
-	hasher            = MakeHashFunc("SHA3")()
+	testHasher        = MakeHashFunc(SHA3Hash)()
 	zeroAddr          = common.Address{}
 	startBlock        = uint64(4200)
 	resourceFrequency = uint64(42)
 	cleanF            func()
 	domainName        = "føø.bar"
+	safeName          string
 )
 
 func init() {
+	var err error
 	log.Root().SetHandler(log.CallerFileHandler(log.LvlFilterHandler(log.LvlTrace, log.StreamHandler(os.Stderr, log.TerminalFormat(true)))))
```

Comment:
Let's remove this or make the loglevel a command line flag like here: https://github.com/ethersphere/go-ethereum/blob/swarm-network-rewrite-syncer/swarm/storage/common_test.go#L37, the test pollutes the console